libraries/dagster-contrib-chroma/dagster_contrib_chroma/resource.py

```diff
@@ -0,0 +1,141 @@
+from contextlib import contextmanager
+from typing import Any, Dict, Union, Generator
+from pydantic import Field
+
+import chromadb
+import chromadb.api
+import chromadb.config
+
+from dagster import ConfigurableResource
+from dagster._utils.backoff import backoff
+
+from .config import LocalConfig, HttpConfig
+
+
+class ChromaResource(ConfigurableResource):
+    """Resource for interacting with a Chroma database.
+
+    Examples:
+        .. code-block:: python
+
+        from dagster import Definitions, asset
+        from dagster_contrib_chroma import ChromaResource, LocalConfig
+
+        @asset
+        def my_table(chroma: ChromaResource):
+            with chroma.get_client() as chroma_client:
+                collection = chroma_client.get_collection("fruits")
+
+                results = collection.query(
+                    query_texts=["hawaii"],
+                    n_results=1,
+                )
```

Comment:
Both of these asset definitions look exactly the same; I think instead we should just option which config we use